modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/js/util/evaluation.es.js

```diff
@@ -136,6 +142,7 @@ export const mergePages = (
 					sourceField.displayErrors || field.fieldName === fieldName,
 				editingLanguageId,
 				valid: field.valid !== false,
+				value: value,
```

Comment:
@rijema and @linolaoj, please use a ternary operator here to save the declaration of variable `value`.

Reply:
@rijema is going to resend